### PR TITLE
Rename test.async to test.cb

### PR DESCRIPTION
**Pull request by jamestalmage**
I think I got everything.

### CODE REVIEW

**Comment by jamestalmage** [comment]
Nope, missed a couple string constants. Everything should be good now.

File: readme.md [hunk 1]
```diff
@@ -433,7 +437,7 @@ test(async t => {
 AVA comes with builtin support for [observables](https://github.com/zenparsing/es-observable).
 If you return an observable from a test, AVA will automatically consume it to completion before ending the test.
 
-*You don't have to use legacy `test.async` mode or `t.end()`.*
+*You don not need to use "callback mode" or call `t.end()`.*
```
Comment (sindresorhus) [inline readme.md]: Typo

**Comment by sindresorhus** [comment]
Looks good when typo is fixed. Just merge.

File: readme.md (hunk 1 above)
Comment (jamestalmage) [inline readme.md]: I am not seeing it.

Comment (jamestalmage) [inline readme.md]: oh wait - yes i do.

**Pull request merged**